astarte/device/device_mqtt.py

```diff
@@ -517,8 +559,36 @@ def __setup_subscriptions(self) -> None:
             f"{self.__get_base_topic()}/control/consumer/properties", qos=2
         )
         for interface in self._introspection.get_all_server_owned_interfaces():
+            self.__subscribe_interface(interface)
```

Comment:
To consolidate the point i made below here we don't need the if check since we are already getting only the server owned interfaces.